test/CurveYieldStrategy.spec.ts

```diff
@@ -506,7 +500,7 @@ describe('CurveYieldStrategy', () => {
       ).to.be.true;
     });
 
-    it('should deduct rage fee (10% which can be changed)', async () => {
+    it.skip('should deduct rage fee (10% which can be changed)', async () => {
```

Comment:
Is the `.skip` intended? Since it's skipped during changes on audited code, we should mention the context either around it or in the commit message.

Reply:
Update: protodev would add a comment for now explaining the reason, and later will update this test case.